trunk/SUAVE/Methods/Center_of_Gravity/compute_fuel_center_of_gravity_longitudinal_range.py

```diff
@@ -100,9 +100,9 @@ def compute_fuel_center_of_gravity_longitudinal_range(vehicle):
     max_cg      = np.zeros_like(fuel_masses)
     
     tank_masses_front_to_back = tank_masses
-    tank_masses_back_to_front = np.flip(tank_masses)
+    tank_masses_back_to_front = np.flip(tank_masses,0)
```

Comment:
what is this 0 for?

Reply:
not sure who/how this was changed, changed it back

Reply:
The 0 is needed when using np.flip as an axis for flipping